llvm/utils/TableGen/IntrinsicEmitter.cpp

```diff
@@ -255,7 +255,7 @@ void IntrinsicEmitter::EmitIntrinsicToOverloadTable(
 static constexpr uint8_t OTable[] = {
   0
   )";
-  for (auto [I, Int] : enumerate(Ints)) {
+  for (const auto& [I, Int] : enumerate(Ints)) {
```

Comment:
enumerate already provides a reference.

Reply:
I see, I have removed the reference in commit https://github.com/llvm/llvm-project/pull/110462/commits/a6e6e75ceba9ae37138a9364ecd4b38a22f30644.